pkg/instance/repository.go

```diff
@@ -229,58 +227,34 @@ func groupWithInstances(instancesMap map[string][]*model.Instance, groupMap map[
 
 // TODO: Rename PopulateRelations? Or something else?
```

Comment:
this PR seems like a good opportunity to get rid of the TODO - maybe `populateParameters`?